types/cli-progress/index.d.ts

```diff
@@ -7,12 +7,13 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 /// <reference types="node" />
 
-import EventEmitter = require("events");
+import EventEmitter = require('events');
 
 export interface Params {
     progress: number;
     eta: number;
     startTime: number;
+    stopTime: number | null;
```

Comment:
```suggestion
    stopTime: number;
```
as with `startTime`, the default options are skipped in the interface (intentionally IMO)
https://github.com/npkgz/cli-progress/blob/e2347a35a6c0922cfb0a077cf5ed21696fba46da/lib/generic-bar.js#L32

Reply:
I believe with `startTime`, the idea was that if the progress isn't "started" (i.e. `startTime = null`), the formatting function won't run and therefore the null can't really be observed. that wouldn't be true about the `stopTime` though, as the formatting function runs both before and after the progress stops.